test/loggerApi_test.cpp

```diff
@@ -706,6 +707,19 @@ void LoggerApiTest::testSampleData2() {
                         getSampleResponse(requestJson2));
 }
 
+void LoggerApiTest::testHeartBeat(){
+    resetTicks();
+    incrementTick();
```

Comment:
Works but seems sketchy and prone to breaking.  Any way you can expose the ability to set the uptime value as part of the tests.  Then it makes the test more sane.